static/panes/diff.js

```diff
@@ -126,94 +120,104 @@ function Diff(hub, container, state) {
-                    this.lhs.refresh();
-                } else {
-                    this.rhs.difftype = parseInt(value);
-                    this.rhs.refresh();
-                }
-                this.updateState();
-            }, this),
-        });
-
-        if (picker.classList.contains('lhsdifftype')) {
-            this.selectize.lhsdifftype = instance;
-        } else {
-            this.selectize.rhsdifftype = instance;
-        }
-
-    }, this));
-
-
-    this.domRoot[0].querySelectorAll('.diff-picker').forEach(_.bind(function (picker) {
-        var instance = new TomSelect(picker, {
-            sortField: 'name',
-            valueField: 'id',
-            labelField: 'name',
-            searchField: ['name'],
-            options: [],
-            items: [],
-            render: {
-                option: function (item, escape) {
-                    var origin = item.editorId !== false ? 'Editor #' + item.editorId : 'Tree #' + item.treeId;
-                    return '<div>' +
-                        '<span class="compiler">' + escape(item.compiler.name) + '</span>' +
-                        '<span class="options">' + escape(item.options) + '</span>' +
-                        '<ul class="meta">' +
-                        '<li class="editor">' + escape(origin) + '</li>' +
-                        '<li class="compilerId">' + escape(getItemDisplayTitle(item)) + '</li>' +
-                        '</ul></div>';
+                dropdownParent: 'body',
+                plugins: ['input_autogrow'],
+                onChange: _.bind(function (value) {
+                    if (picker.classList.contains('lhsdifftype')) {
+                        this.lhs.difftype = parseInt(value);
+                        this.lhs.refresh();
+                    } else {
+                        this.rhs.difftype = parseInt(value);
+                        this.rhs.refresh();
+                    }
+                    this.updateState();
+                }, this),
+            });
+
+            if (picker.classList.contains('lhsdifftype')) {
+                this.selectize.lhsdifftype = instance;
+            } else {
+                this.selectize.rhsdifftype = instance;
+            }
+        }, this)
+    );
+
+    this.domRoot[0].querySelectorAll('.diff-picker').forEach(
+        _.bind(function (picker) {
+            var instance = new TomSelect(picker, {
+                sortField: 'name',
+                valueField: 'id',
+                labelField: 'name',
+                searchField: ['name'],
+                options: [],
+                items: [],
+                render: {
+                    option: function (item, escape) {
+                        var origin = item.editorId !== false ? 'Editor #' + item.editorId : 'Tree #' + item.treeId;
+                        return (
+                            '<div>' +
+                            '<span class="compiler">' +
```

Comment:
This choice from prettier is a little weird but no major concern

Reply:
Agreed; I think I'll leave this as-is. I've seen worse, and frankly we could phrase this a lot better now we're transpiling all the things too. But ... for another PR!

Reply:
Good point!